package.json

```diff
@@ -17,9 +17,11 @@
   },
   "scripts": {
     "app": "npm run l10n && gulp app",
-    "build": "gulp",
-    "l10n": "pontoon-to-json",
+    "build": "npm run lint && gulp",
```

Comment:
Having a test as part of the build task seems a bit weird to me. Won't Travis run the linting task twice this way?